### PR TITLE
docs: lead with the engine in the README, move the direction to VISION.md, add support levels and area stewards

### PR DESCRIPTION
**Pull request by Fmarzochi**
## What Changed

- **README** leads with the engine: memory, Session Mesh, Guardian and Token Crusher, each with its own section, in that order. The banner, the language selector rows and both demos stay exactly where they were. The prompt library moves to an optional section at the end, with `egc install --prompt-library` and the per-tool profile as the two ways to add it. The Support section states that the engine is Apache-2.0 and stays free, and links the new vision document.
- **`docs/VISION.md`** (new) holds the direction: what EGC is, the five principles (engine first, local first, one package, honest integrations, free engine), where EGC is going (Agent Memory Interchange, Team Brain, public savings, one-command onboarding), sustainability, and the milestones and non-goals moved verbatim from the end of `docs/ROADMAP.md`. The roadmap is now the release-by-release record and points at the vision.
- **`docs/governance/support-levels.md`** (new) separates what the maintainers guarantee per tool from how each tool integrates: core-supported (Claude Code, Antigravity, Cursor, Codex CLI, OpenCode) with the criteria applied, community-supported (the other fifteen), retired (the three from 2026-08-16), and how a tool moves between levels. Nothing leaves the package by a level change.
- **`docs/governance/stewards.md`** (new) opens area stewardship as a role without access: what a steward does, how someone becomes one, and the first three invitations (Windows and OpenCode, Session Mesh, Installer). The invitations themselves go out after this merges.
- Pointers: `docs/README.md`, `docs/governance/README.md`, `docs/MAINTAINERS.md` and `docs/spec/integration-tiers.md`. CHANGELOG entry under Unreleased.

## Why This Change

Every reader of the README today meets the prompt library before understanding the engine, and the roadmap mixes the strategic direction with the release record. The engine is what EGC is; the library rides on it. This round makes the documents say so, gives the direction one short home, and states in writing what has only been true in practice: which tools the maintainers stand behind, and that the engine stays free.

## Testing Done

- [x] Manual testing completed: banner, selector rows and both demo images present and unchanged; no forbidden dash forms in the added text.
- [x] Automated tests pass locally (`node tests/run-all.js`): 151 suites, 4834 tests, all passing (the validators suite needed the README provider sentence kept in the form the LLM providers validator reads; fixed before this commit and rerun: 170/170)
- [x] Edge cases considered and tested: the catalog parity sentence (`access to 61 agents, 232 skills, and 77 commands, plus 109 rules`), the installer count regex and the `20 AI coding tools` phrase are all kept; `tests/ci/catalog.test.js` 7/7, `tests/spec/integration-tiers.test.js` 5/5, `tests/scripts/install-ps1.test.js` count check green.
- [x] Not applicable: no file shared across concurrent EGC processes is touched.

## Type of Change
- [ ] `fix:` Bug fix
- [ ] `feat:` New feature
- [ ] `refactor:` Code refactoring
- [x] `docs:` Documentation
- [ ] `test:` Tests
- [ ] `chore:` Maintenance/tooling
- [ ] `ci:` CI/CD changes

## Security & Quality Checklist
- [x] No secrets or API keys committed (ghp_, sk-, AKIA, xoxb, xoxp patterns checked)
- [x] JSON files validate cleanly (none changed)
- [x] Shell scripts pass shellcheck (none changed)
- [x] Pre-commit hooks pass locally
- [x] No sensitive data exposed in logs or output
- [x] Follows conventional commits format

## Documentation
- [x] Updated relevant documentation
- [x] Added comments for complex logic (none needed)
- [x] README updated

---
## Summary by cubic
Restructures the docs so the README leads with the engine, moves the project direction to `docs/VISION.md`, and codifies support levels and area stewardship.

- README presents `memory`, Session Mesh, Guardian, and Token Crusher first; the prompt library is an optional section with the install flag.
- `docs/VISION.md` holds the principles, milestones, and non-goals; the roadmap is now the release-by-release record, the changelog and roadmap record the Turkish and German translations under v1.1.16, and the vision's translation note no longer points at a document for each shipped language.
- `docs/governance/support-levels.md` defines core-supported vs community-supported tools and how tools move between levels.
- `docs/governance/stewards.md` opens area stewardship as a review and triage role without elevated access; invited candidates are listed with an `invited` status until they accept.
- Indexes and maintainers docs point to the new documents.

<sup>Written for commit 936e379fac5ef9f377eb93a0a1dcc74c07f691fa. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/Fmarzochi/EGC/pull/1441?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

**Comment by Fmarzochi** [comment]
Two of the four findings are about #1440, not this PR: the `--prompt-library` flag and the default of no ship in [#1440](https://github.com/Fmarzochi/EGC/pull/1440) (bash and PowerShell installers, parser, docs, tests), which merges first; this PR then takes main and the README describes what the installer does. The other two are taken: the stewards text now says invited candidates are listed with an `invited` status until they accept, and the translations bullet moved from the roadmap says v1.1.15 for the four languages, which is what the changelog records.

File: README.md [hunk 1]
```diff
@@ -40,54 +38,56 @@ npm install -g @egchq/egc && egc install
...
+## Prompt Library (Optional)
 
-A live dashboard displaying agent activity, tokens, and costs spins up automatically right after installation. Prefer manual control? Every command is documented in the [installation guide](docs/installation.md): you will probably never need to type one.
+Separate from the engine, and off by default, EGC also ships a library written from real engineering sessions: you get access to 61 agents, 232 skills, and 77 commands, plus 109 rules. Specialists that review your code on their own, best-practice guides for every language and situation, shortcuts that run a whole sequence of tasks, and style rules that keep your code consistent. Add it to every detected tool with `egc install --prompt-library`, or to one tool with `egc install --target <tool> --profile full`. Skip it and the engine works exactly the same.
```
[the remark this answers is not in the document]
Comment (Fmarzochi) [reply README.md:77]: Landed on main with #1440 (11eb5683): the flag exists in the parser and both installers, and the interactive default is no. This branch carries that main now (010d1c9a).

File: README.md [hunk 2]
```diff
@@ -28,9 +28,7 @@ Every conversation builds your project's collective intelligence. Every agent in
 npm install -g @egchq/egc && egc install
 ```
 
-- **Reduce context waste by up to 90%, cut token costs, and keep every AI perfectly aligned across sessions.**
-- **Guardian: validate every command before execution and block dangerous writes. Every shared brain comes with a built-in safety layer.**
-- **One command, zero config: memory stays local and encrypted on your machine, and never gets committed to git.**
+That is the whole engine. `egc install` detects the tools you have, registers the two local MCP servers in each of them, writes the memory protocol every agent reads, and sets up the Token Crusher. It asks one question, whether you also want the optional prompt library, and the default is no.
```
[the remark this answers is not in the document]
Comment (Fmarzochi) [reply README.md:31]: Landed on main with #1440 (11eb5683): the flag exists in the parser and both installers, and the interactive default is no. This branch carries that main now (010d1c9a).

File: docs/VISION.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+- Shared state between team members (multi-user installations beyond git-backend team memory)
+- Organization-level installations and role-based context scoping
+- Cross-project memory federation
+- Stable MCP server API with versioned interfaces
+- `egc-guardian` and `egc-memory` promoted to GA with backward-compatibility guarantees
+
+### v1.3.0: Growth
+
+- Community translations: Ukrainian, Malay (German, French, Italian, and Turkish shipped in v1.1.15)
```
[the remark this answers is not in the document]
Comment (Fmarzochi) [reply docs/VISION.md]: The sentence no longer dates the four languages: it says they already shipped and points at the changelog for the release of each, which is the source of truth here.

File: docs/VISION.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+- Shared state between team members (multi-user installations beyond git-backend team memory)
+- Organization-level installations and role-based context scoping
+- Cross-project memory federation
+- Stable MCP server API with versioned interfaces
+- `egc-guardian` and `egc-memory` promoted to GA with backward-compatibility guarantees
+
+### v1.3.0: Growth
+
+- Community translations: Ukrainian, Malay (German, French, Italian and Turkish already shipped; the changelog has the release of each)
```
[the remark this answers is not in the document]
Comment (Fmarzochi) [reply docs/VISION.md]: Taken, with one difference: the sentence now points at no document at all. The roadmap dates German and Turkish in v1.1.15 while the tags put #970 and #972 in v1.1.16, so neither file is a safe pointer from here; the note just says the four languages already shipped. The roadmap line gets its own correction separately.

File: docs/ROADMAP.md [hunk 5]
```diff
@@ -221,13 +221,13 @@ Closes the commit-privacy scope started in v1.1.12:
...
 - Dashboard serves static files added after startup without a restart, with a symlink guard and a debounced manifest rebuild that preserves the traversal protection (#928, @hanu-14)
 - French README translation, the 11th language, wired into every language selector (#948, @ankit24417-sys)
-- Turkish README translation, the 12th language, wired into every language selector (#970, @Iahmacun)
-- German README translation, the 13th language, wired into every language selector (#972, @rathaur-ankit)
 - High-severity advisories cleared: fast-uri and linkify-it across the root and mcp-server lockfiles (#967), and `@hono/node-server` forced to 2.x via npm override to close the last Dependabot and Scorecard findings (#968)
 - ClusterFuzzLite weekly schedule disabled while upstream is dead-locked for JavaScript, keeping manual `workflow_dispatch` (EGC#910)
 
 ## v1.1.16: Security Hardening and Crowdin Root Fix (Released 2026-07-26)
 
+- Turkish README translation, the 12th language, wired into every language selector (#970, @Iahmacun)
```
[the remark this answers is not in the document]
Comment (Fmarzochi) [reply docs/ROADMAP.md:229]: Taken: the 1.1.16 section of the changelog now records the Turkish (#970) and German (#972) translations, which is where the tags place them (v1.1.16 is the first tag containing both merges). Roadmap and changelog agree.
